error on web

## Description

Console Error
Unexpected text node: ). A text node cannot be a child of a <View>.
lib/console-log-buffer.ts (56:15)

Open

 
 54 |
   console
.
error 
=
 (
...
args
:
 unknown[]) 
=>
 {
 
 55 |
     addEntry(
"error"
,
 args)
;
>
 56 |
     origError
.
apply(console
,
 args)
;
 
    |
               
^
 
 57 |
   }
;
 
 58 |
 }
 
 59 |
components/ui/button.tsx (355:13)

Open

 
 353 |
  
<
/
View
>
 
 354 |
) 
:
 (
>
 355 |
  
<
View
 
     |
  
^
 
 356 |
    style
=
{{ flexDirection
:
 
'row'
,
 alignItems
:
 
'center'
,
 gap
:
 
6
 }}
 
 357 |
  
>
 
 358 |
    {icon 
&&
 (
Component Stack
40

forwardRef$argument_0
components/ui/button.tsx:355:13
FeedbackCard
app/(tabs)/settings.tsx:253:11
Settings
app/(tabs)/settings.tsx:180:9
WrappedScreenComponent
node_modules/expo-router/build/useScreens.js:194:16

## Diagnostic Info

- App Version: 0.7.1
- Platform: web
- OS Version: 0.0.0

## Device Info

- Device: Macintosh
- Device Name: unknown
- Total Memory: 32768 MB
- Execution Environment: bare

## Recent Interactions

1. [2026-04-19T15:48:45.387Z] navigate: /settings
2. [2026-04-19T15:48:43.644Z] navigate: /errors
3. [2026-04-19T15:48:24.583Z] navigate: /settings
4. [2026-04-19T06:10:56.087Z] navigate: /
5. [2026-04-19T06:10:50.750Z] navigate: /session/5272016d-49e7-4a29-805e-fa3cbd4f3349
6. [2026-04-19T06:10:45.022Z] navigate: /
7. [2026-04-19T06:10:37.108Z] navigate: /settings
8. [2026-04-19T06:10:35.408Z] navigate: /progress
9. [2026-04-19T06:10:30.582Z] navigate: /nutrition
10. [2026-04-19T06:10:29.374Z] navigate: /nutrition/targets
11. [2026-04-19T06:10:28.337Z] navigate: /nutrition
12. [2026-04-19T06:10:25.539Z] navigate: /nutrition/templates
13. [2026-04-19T06:10:22.382Z] navigate: /nutrition
14. [2026-04-19T06:10:20.333Z] navigate: /exercises
15. [2026-04-18T06:53:01.871Z] navigate: /settings
16. [2026-04-18T06:52:52.868Z] navigate: /nutrition
17. [2026-04-18T06:52:17.292Z] navigate: /nutrition/add
18. [2026-04-18T06:52:15.269Z] navigate: /nutrition
19. [2026-04-18T06:52:13.463Z] navigate: /settings
20. [2026-04-18T06:52:12.005Z] navigate: /errors
21. [2026-04-18T06:52:10.469Z] navigate: /settings
22. [2026-04-18T06:52:09.216Z] navigate: /progress
23. [2026-04-18T06:51:57.506Z] navigate: /settings
24. [2026-04-18T06:51:55.389Z] navigate: /progress
25. [2026-04-18T06:51:51.334Z] navigate: /nutrition
26. [2026-04-18T04:58:48.380Z] navigate: /exercises
27. [2026-04-18T04:58:46.600Z] navigate: /nutrition
28. [2026-04-18T04:58:45.105Z] navigate: /nutrition/add
29. [2026-04-18T04:58:38.627Z] navigate: /nutrition
30. [2026-04-18T04:58:35.666Z] navigate: /progress
31. [2026-04-18T04:58:16.964Z] navigate: /settings
32. [2026-04-18T04:42:28.013Z] navigate: /
33. [2026-04-18T04:42:25.949Z] navigate: /onboarding/recommend
34. [2026-04-18T04:42:22.971Z] navigate: /onboarding/setup

## Error Log

No errors recorded

## Console Logs

1. [2026-04-19T15:48:18.466Z] [WARN] Route "./screen-config.ts" is missing the required default export. Ensure a React component is exported as default.
2. [2026-04-19T15:48:19.090Z] [WARN] props.pointerEvents is deprecated. Use style.pointerEvents
3. [2026-04-19T15:48:19.239Z] [WARN] `flexWrap: `wrap`` is not supported with the `VirtualizedList` components.Consider using `numColumns` with `FlatList` instead.
4. [2026-04-19T15:48:24.544Z] [ERROR] Unexpected text node: Errors (. A text node cannot be a child of a <View>.
5. [2026-04-19T15:48:24.547Z] [ERROR] Unexpected text node: ). A text node cannot be a child of a <View>.
6. [2026-04-19T15:48:24.569Z] [ERROR] Unexpected text node: Errors (. A text node cannot be a child of a <View>.
7. [2026-04-19T15:48:24.569Z] [ERROR] Unexpected text node: ). A text node cannot be a child of a <View>.
8. [2026-04-19T15:48:24.602Z] [ERROR] Unexpected text node: Errors (. A text node cannot be a child of a <View>.
9. [2026-04-19T15:48:24.602Z] [ERROR] Unexpected text node: ). A text node cannot be a child of a <View>.
10. [2026-04-19T15:48:43.629Z] [ERROR] Unexpected text node: Errors (. A text node cannot be a child of a <View>.
11. [2026-04-19T15:48:43.629Z] [ERROR] Unexpected text node: ). A text node cannot be a child of a <View>.
12. [2026-04-19T15:48:45.374Z] [ERROR] Unexpected text node: Errors (. A text node cannot be a child of a <View>.
13. [2026-04-19T15:48:45.374Z] [ERROR] Unexpected text node: ). A text node cannot be a child of a <View>.
14. [2026-04-19T15:49:01.043Z] [ERROR] Unexpected text node: Errors (. A text node cannot be a child of a <View>.
15. [2026-04-19T15:49:01.043Z] [ERROR] Unexpected text node: ). A text node cannot be a child of a <View>.

--- a/__tests__/acceptance/settings.test.tsx
+++ b/__tests__/acceptance/settings.test.tsx
@@ -435,6 +435,12 @@ describe('Settings Screen Acceptance', () => {
     })
   })
 
+  it('Error log button renders count as single text node (no View-wrapped text)', async () => {
+    const { findByText } = renderScreen(<Settings />)
+    const label = await findByText('Errors (2)')
+    expect(label).toBeTruthy()
+  })
+
   // ── Import from Strong Button ──────────────────────────────────
 
   it('Import from Strong button navigates to import screen', async () => {

--- a/app/(tabs)/settings.tsx
+++ b/app/(tabs)/settings.tsx
@@ -250,7 +250,7 @@ function FeedbackCard({ colors, count, onBug, onFeature, onErrors }: {
         <View style={styles.buttonFlow}>
           <Button variant="default" icon={Bug} onPress={onBug} accessibilityLabel="Report a bug">Report Bug</Button>
           <Button variant="outline" icon={Lightbulb} onPress={onFeature} accessibilityLabel="Request a feature">Feature Request</Button>
-          <Button variant="outline" icon={List} onPress={onErrors} accessibilityLabel={`View error log, ${count} ${count === 1 ? "error" : "errors"}`}>Errors ({count})</Button>
+          <Button variant="outline" icon={List} onPress={onErrors} accessibilityLabel={`View error log, ${count} ${count === 1 ? "error" : "errors"}`}>{`Errors (${count})`}</Button>
         </View>
       </CardContent>
     </Card>

